### ISSUE TITLE
Is this project still active?

### ISSUE THREAD

**Issue by bschonec** [devans10/terraform-provider-flash#31]
Both this project an the [unsupported fork](https://github.com/PureStorage-OpenConnect/terraform-provider-flash) are inactive.

**Comment by devans10** [comment]
Sorry. Yes, this project is inactive.  I no longer have access to any hardware to test on.

**Comment by bschonec** [comment]
Would you be willing to transfer ownership over to me?  I have a couple of Flashblades and I find your project very useful.

**Comment by sdodsley** [comment]
@bschonec please note that Pure will not endorse, or support, any Terraform provider from a 3rd party.

**Comment by devans10** [comment]
@bschonec Yes, I would be willing to transfer ownership.  would you want to take the pugo repository as well?
